ngraph_bridge/enable_variable_ops/ngraph_replace_variable_modifiers.cc

```diff
@@ -35,6 +35,33 @@ namespace ngraph_bridge {
 // 1. ReplaceModifiers
 // 2. RewriteForTracking
 
+template <typename T1, typename T2>
```

Comment:
Here are the 3 types of `Input` functions:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/graph/node_builder.h#L91
```
// For inputs that take a single tensor.
  NodeBuilder& Input(Node* src_node, int src_index = 0);
  NodeBuilder& Input(NodeOut src);

  // For inputs that take a list of tensors.
  NodeBuilder& Input(gtl::ArraySlice<NodeOut> src_list);
```

We are surely not using `NodeBuilder& Input(Node* src_node, int src_index = 0);`

We could be using `NodeBuilder& Input(NodeOut src);` or ` NodeBuilder& Input(gtl::ArraySlice<NodeOut> src_list);`

In this case I think we are using only 1 variant, the `NodeBuilder& Input(NodeOut src);`. If we are using only 1 variant, we can remove `template <typename T1, typename T2>`